### PR TITLE
Muzima 422

### PR DESCRIPTION
**Pull request by mssavai**

### CODE REVIEW

File: res/layout-land/activity_imaging.xml [hunk 1]
```diff
@@ -138,7 +138,7 @@
                     android:onClick="chooseImage"
                     android:padding="11dp"
                     android:layout_marginTop="3dp"
-                    android:text="@string/choose_image"
+                    android:text="@string/title_choose_image"
```
Comment (sthaiya) [inline res/layout-land/activity_imaging.xml]: Should this be title_image_choose

File: res/layout-land/activity_video.xml [hunk 2]
```diff
@@ -96,7 +96,7 @@
             android:background="@drawable/sel_button_round"
             android:onClick="acceptVideo"
             android:padding="11dp"
-            android:text="@string/accept"
+            android:text="@string/general_accept_text"
```
Comment (sthaiya) [inline res/layout-land/activity_video.xml]: Can we remove the '_text' suffix on all general strings

File: res/layout/item_preference.xml [hunk 3]
```diff
@@ -26,7 +26,7 @@
                 android:layout_gravity="center"
                 android:clickable="true"
                 android:src="@drawable/ic_action_discard"
-                android:contentDescription="@string/delete_text"/>
+                android:contentDescription="@string/text_delete"/>
```
Comment (sthaiya) [inline res/layout/item_preference.xml]: this should be general_delete

File: res/values/strings.xml [hunk 4]
```diff
@@ -1,302 +1,115 @@
...
+    <string name="info_client_incomplete_form_unavailable">No incomplete forms</string>
+    <string name="info_clients_local_search_unmatched">No clients matched your search locally. Enter at least 3 characters for
+        remote search.
+    </string>
+    <string name="info_clients_remote_search_unmatched">No clients matched your search on server</string>
+    <string name="info_cohort_prefixes_unavailable">No Cohort Prefixes Added</string>
+    <string name="info_cohorts_metadata_unavailable">No Cohorts Available</string>
+    <string name="info_complete_form_unavailable">No completed and non synced forms on device</string>
+    <string name="info_concepts_unavailable">No concept added yet.</string>
+    <string name="info_crash_dialog_ok_click">Thank you!</string>
```
Comment (sthaiya) [inline res/values/strings.xml]: Can the resource be called general_thank_you so that it can be reused

**Comment by sthaiya** [comment]
Some comments to address

**Comment by mssavai** [comment]
The comments have now been addressed.

File: res/layout/activity_video.xml [hunk 5]
```diff
@@ -1,146 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (sthaiya) [inline res/layout/activity_video.xml]: This is needed for layout portrait, so is imaging and audio

**Pull request merged**